app/assets/javascripts/dashboard/current_tasks.js

```diff
@@ -0,0 +1,83 @@
+/* global dropdownSelector */
+/* eslint-disable no-param-reassign */
+
+var DasboardCurrentTasksWidget = (function() {
+  function initFilters() {
+    var sortFilter = '.curent-tasks-filters .sort-filter';
+    var viewFilter = '.curent-tasks-filters .view-filter';
+    var projectFilter = '.curent-tasks-filters .project-filter';
+    var experimentFilter = '.curent-tasks-filters .experiment-filter';
+
+    $('.curent-tasks-filters .clear-button').click(() => {
```

Comment:
Probably here also first two filters should go to default state? Maybe ask product or UX team.